src/cli/README.md

```diff
@@ -393,6 +393,16 @@ fd41:2650:a6f5:0::/64
 Done
 ```
 
+### br nat64prefix
+
+Get the local nat64 prefix of the Border Router.
```

Comment:
Add a comment that this requires `OPENTHREAD_CONFIG_BORDER_ROUTING_NAT64_ENABLE`?

Reply:
Updated